binaries/cli/src/template/python/listener/listener-template.py

```diff
@@ -0,0 +1,11 @@
+from dora import Node
+import pyarrow as pa
+
+node = Node()
+
+event = node.next()
+if event["type"] == "INPUT":
+    message = event["value"][0].as_py()
+    print(
+        f"""I heard {message} from {event["id"]}"""
+    )
```

Comment:
Could we make this a loop using:
```suggestion
for event in node:
    if event["type"] == "INPUT":
        message = event["value"][0].as_py()
        print(
            f"""I heard {message} from {event["id"]}"""
        )
```